tests/integration/test_secure_message.py

```diff
@@ -45,13 +47,24 @@ def tearDown(self):
     def test_get_thread_success(self, mock_request):
         mock_request.get(url_get_thread, json={'messages': [conversation_json], 'is_closed': False})
         mock_request.get(url_get_conversation_count, json={'total': 0})
+        mock_request.get(url_get_survey_long_name, json={
+                    "id": "02b9c366-7397-42f7-942a-76dc5876d86d",
+                    "shortName": "QBS",
+                    "longName": "Quarterly Business Survey",
+                    "surveyRef": "139",
+                    "legalBasis": "Statistics of Trade Act 1947",
+                    "surveyType": "Business",
+                    "surveyMode": "EQ",
+                    "legalBasisRef": "STA1947"
+                })
 
         response = self.app.get("secure-message/threads/9e3465c0-9172-4974-a7d1-3a01592d1594",
                                 headers=self.headers, follow_redirects=True)
         self.assertEqual(response.status_code, 200)
         self.assertTrue('Peter Griffin'.encode() in response.data)
         self.assertTrue('testy2'.encode() in response.data)
         self.assertTrue('something else'.encode() in response.data)
```

Comment:
This is missing tests checking for the survey name and business name which have been added to the page

Reply:
survey name added to test